docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,820 @@
+# Slash Commands
+
+Slash Commands are the new, exciting way to build and interact with apps on Discord.
+
+With Slash Commands, all you have to do is type `/` and you're ready to use your favorite bot. Users can learn everything your bot does and easily find new features as you add them. Validation, error states, and helpful UI walks them through your commands, meaning they can get it right the first time, especially on mobile. You now have one more ally in the fight against your phone's autocorrect.
+
+Slash Commands set your users up for success instead of confusion and frustration. They separate how users think and how your code works, meaning no matter how complex your codebase and commands may get, people who love your bot will find it approachable and easy to use.
+
+Let's get started!
+
+## A Quick Note on Limits
+
+In this documentation you'll find some notes about limits and caps on certain parts of Slash Commands. At a high level, they are as follows:
+
+- An app can have up to 50 global commands
```

Comment:
what counts towards this limit? 50 top level commands or, do groups and subcommands count towards this as well?

Reply:
50 top-level commands. I think there's a max of subcommands and groups that should be documented as well